File: src/ru/alexpt/test/my25java/TopLevelDemo.java
package ru.alexpt.test.my25java;

import java.awt.*;
import java.awt.event.*;
import javax.swing.*;

public class TopLevelDemo {

    private static void createAndShowGUI() {
        // Создание и настройка окна
        JFrame frame = new JFrame("TopLevelDemo");
        // Настройка закрытия приложения после закрытия окна
        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);

        // Создать строку меню MenuBar. Сделать зеленый фон
        JMenuBar greenMenuBar = new JMenuBar();
        greenMenuBar.setOpaque(true);
        greenMenuBar.setBackground(new Color(154, 165, 127));
        greenMenuBar.setPreferredSize(new Dimension(200, 20));

        // Создать желтую метку для размещения на панели содержимого
        JLabel yellowLabel = new JLabel();
        yellowLabel.setOpaque(true);
        yellowLabel.setBackground(new Color(248, 213, 131));
        yellowLabel.setPreferredSize(new Dimension(200, 180));

        // Установить строку меню и добавить метку на панель содержимого
        frame.setJMenuBar(greenMenuBar);
        frame.getContentPane().add(yellowLabel, BorderLayout.CENTER);

        // Отобразить окно
        frame.pack();
        frame.setVisible(true);
    }

    public static void main(String[] args) {
        // Запланировать задание для потока диспетчеризации событий:
        // создание и показ этого приложения с GUI
        javax.swing.SwingUtilities.invokeLater(new Runnable() {
            public void run() {
                createAndShowGUI();
            }
        });
    }
}

File: src/ru/alexpt/test/my25kt/TopLevelDemo.kt
package ru.alexpt.test.my25kt

import java.awt.*
import java.awt.event.*
import javax.swing.*

fun main(args: Array<String>) {
    visible()
}

fun createAndShowGUI() {
        // Создание и настройка окна
        val frame = JFrame("TopLevelDemo")
        // Настройка закрытия приложения после закрытия окна
        frame.defaultCloseOperation = JFrame.EXIT_ON_CLOSE

        // Создать строку меню MenuBar. Сделать зеленый фон
        val greenMenuBar = JMenuBar()
        greenMenuBar.isOpaque = true
        greenMenuBar.background = Color(154, 165, 127)
        greenMenuBar.preferredSize = Dimension(200, 20)

        // Создать желтую метку для размещения на панели содержимого
        val yellowLabel = JLabel()
        yellowLabel.isOpaque = true
        yellowLabel.background = Color(248, 213, 131)
        yellowLabel.preferredSize = Dimension(200, 180)

        // Установить строку меню и добавить метку на панель содержимого
        frame.jMenuBar = greenMenuBar
        frame.contentPane.add(yellowLabel, BorderLayout.CENTER)

        // Отобразить окно
        frame.pack()
        frame.isVisible = true
}

fun visible(){
    // Запланировать задание для потока диспетчеризации событий:
    // создание и показ этого приложения с GUI
    javax.swing.SwingUtilities.invokeLater { createAndShowGUI() }
}